sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AssistantsClientTestBase.java

```diff
@@ -214,18 +213,6 @@ void createRunRunner(Consumer<CreateRunOptions> testRunner, String assistantId)
         testRunner.accept(new CreateRunOptions(assistantId));
     }
 
-    void createRetrievalRunner(BiConsumer<FileDetails, AssistantCreationOptions> testRunner) {
```

Comment:
Shouldn't we have repurposed this runner for `file_search` instead? I believe that the tool definition was only renamed but functionally is nearly identical.